samples/wifi/monitor/README.rst

```diff
@@ -0,0 +1,110 @@
+.. _wifi_monitor_sample:
+
+Wi-Fi: Monitor
```

Comment:
The way I see it, this sample should be a raw capture sample which can be configured in monitor mode and (later) promiscuous mode . Let's discuss this internally and modify accordingly.

Reply:
Following internal discussions, maintain "Monitor" as a distinct example, and treat "promiscuous mode" as a separate illustration.